packages/cli/CHANGELOG.md

```diff
@@ -507,7 +507,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - support subcommand codegen
 - support subcommand init
 
-[Unreleased]: https://github.com/subquery/subql/compare/cli/4.2.2...HEAD
+[Unreleased]: https://github.com/subquery/subql/compare/cli/4.2.3...HEAD
```

Comment:
Wheres the rest of the changelog changes? Are you using `yarn changelog:release`?